File: apps/fast-api-server/src/routes/health.py
from fastapi import APIRouter
from pydantic import BaseModel, Field

router = APIRouter()

# =============================================================================
# Schemas
# =============================================================================


class HealthResponse(BaseModel):
    """ヘルスチェックレスポンスモデル"""

    message: str = Field(..., description="ステータスメッセージ")
    app_version: str | None = Field(None, description="アプリケーションバージョン")


# =============================================================================
# Endpoints
# =============================================================================


@router.get(
    path="",
    summary="APIサーバーヘルスチェック",
)
def health() -> HealthResponse:
    """APIサーバーヘルスチェックエンドポイント"""
    return HealthResponse(message="alive", app_version="1.0.0")

File: apps/hono-server/src/routes/health.ts
import { Hono } from 'hono'
import type { Env } from '../types/env'

const health = new Hono<{ Bindings: Env }>()

health.get('/', (c) => {
  return c.json({ message: 'alive', app_version: '1.0.0' })
})

export default health
